Tools/Scripts/hooks/pre-push

```diff
@@ -295,7 +295,7 @@ def main(name, remote):
             print_error("'{}' comes from an uncategorized remote, it's not safe to push it publically".format(commit))
             return 1
         # Security level of source exceeds security level of target
-        if categorization_class == 1 and security_level > (target_security_level or 0):
+        if categorization_class == 1 and (MAX_LEVEL if security_level is None else security_level) > (target_security_level or 0):
```

Comment:
Would this be better as:
```
if categorisation_class == 1 and (security_level or MAX_LEVEL) > (target_security_level or 0):
```

in keeping with the existing `(target_security_level or 0)`?

Reply:
The trouble here is that we don't want to promote the `security_level` to `MAX_LEVEL` if the `security_level` is indeed `0` (which is not only possible, but going to be the case most of the time, since `github.com:WebKit/WebKit` has a security level of `0`)